rpms/SPECS/4.0.0/wazuh-manager-4.0.0.spec

```
@@ -235,13 +235,11 @@ if [ $1 = 2 ]; then
 
   # Delete 3.X Wazuh API service
   if [ "$MAJOR" = "3" ] && [ -d %{_localstatedir}/api ]; then
-    if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1; then
+    if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 && systemctl is-active --quiet wazuh-agent > /dev/null 2>&1; then
```

```suggestion
    if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1; then
```
This check must be done always because the `wazuh-api` package is no longer installed.